scripts/start.sh

```
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+# Get the absolute path to the project root
+PROJECT_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
```

Shellcheck

[warning] In POSIX sh, BASH_SOURCE is undefined.

(SC3028)